keyboards/portal_66/readme.md

```diff
@@ -0,0 +1,16 @@
+# Portal 66
```

Comment:
Could you add instructions on how to enter the bootloader?

Reply:
Done ✅